inmoose/cohort_qc/cohort_metric.py

```diff
@@ -85,7 +85,7 @@ def __init__(
         batch_column: str,
         data_expression_df: pd.DataFrame,
         data_expression_df_before: pd.DataFrame = None,
-        covariates: list = None,
+        covariates: list = [],
```

Comment:
I think Antoine told us to avoid mutable types in default values

https://zestedesavoir.com/billets/2204/bizarreries-pythoniques-arguments-par-defaut-mutables/

Reply:
Indeed.
If you ask me, this is a Python flaw: the behavior of a default argument should not depend on its type...
Or maybe it's just my C++ side taking over...

Anyhow, I'll keep the `None` default value, and add a `if covariates is None: covariates = []` statement